roles/ceilometer-common/defaults/main.yml

```diff
@@ -62,3 +62,5 @@ ceilometer:
     debug: False
     verbose: True
   cafile: "{{ ssl.cafile|default('/etc/ssl/certs/ca-certificates.crt') }}"
+  sensu_criticality:
+    check_ceilometer_api: 'critical'
```

Comment:
this should probably be something like `ceilometer.monitoring.sensu_checks.ceilometer_api.severity`   that way if we later want to be able to add other vars like "enabled: true:false" to a given task it will be easy to extend.

obvs same applies to rest of checks.